### PR TITLE
rl-2311: Link to blog post on the file set library

### PR DESCRIPTION
**Pull request by infinisil**
## Description of changes

Now that https://www.tweag.io/blog/2023-11-28-file-sets/ is published we can link to it from the release notes.

The release managers @RaitoBezarius and @figsoda already confirmed this would be okay to have [^1] [^2], but @figsoda wanted to give the final say to the release editors, so I requested them for a review here.

This work is sponsored by [Antithesis](https://antithesis.com/) :sparkles:

[^1]: https://matrix.to/#/!aGqRytqbCECitOFhbt:nixos.org/$KJY-rzpMurM2VQbgk449nA_P1KgeqZZJEWWhjC-JWuw
[^2]: https://matrix.to/#/!aGqRytqbCECitOFhbt:nixos.org/$94sRBiLb2zirG1ZiRyi7PeYn3UBXhhtYWp1oD7gN8S0

## Things done

- [x] Rendered the manual and checks it looks correct.

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by riotbib — APPROVED** [review]
Linking directly to a commercial site works for me in this case, since it's a big contribution and we are short on time.

Though we should find a more elegant solution for the future I guess.

I'd like to wait for @alejandrosame for the final approval.

(Congrats to the improvement!)

**Comment by alejandrosame** [comment]
I think it's not intrusive, so it's ok. If we start seeing sponsor or employer links become a norm we could add a shoutout section in the future.

**Review by alejandrosame — APPROVED** [review]

**Pull request merged**
